package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@daostack/client",
-  "version": "0.0.53",
+  "version": "0.0.54",
```

Comment:
.55?

Reply:
aren't we ready for a minor version yet?? :)

Reply:
why not, let's go wild next time!